straxen/plugins/acqmon_processing.py

```diff
@@ -51,21 +57,19 @@ def compute(self, raw_records_aqmon):
 
 @export
 @strax.takes_config(
-    strax.Option('min_veto_gap', default=int(1e6), type=int,
-                 help='Minimum separation between veto stop and start pulses [ns]'),
-    strax.Option('max_veto_gap', default=int(5e8), type=int,
+    strax.Option('max_veto_window', default=int(5e8), track = True, type = int,
                  help='Maximum separation between veto stop and start pulses [ns]'),
-    strax.Option('channel_map', track=False, type=immutabledict,
+    strax.Option('channel_map', type = immutabledict, track=False, 
                  help='immutabledict mapping subdetector to (min, max)'
-                      'channel number.'))
```

Comment:
Not entirely sure but guess you don't have to change this since it used to be exactly the same as in the DAQ-reader:
https://github.com/XENONnT/straxen/blob/c84928fb4ea20bd1872405e9dcef87db9508d220/straxen/plugins/daqreader.py#L61

Reply:
Hmm...I actually wonder if that is the reason why I couldn't set track=True for this option in the aqmon plugin.